lib/tasks/coronavirus.rake

```diff
@@ -1,23 +1,25 @@
-## This rake task deletes all CoronavirusPage from the collections publisher database
-## Once it's been run, visiting https://collections-publisher.publishing.service.gov.uk/coronavirus will
-## trigger the latest data to be pulled from github, and then this task should be deleted.
+# Temporary rake task whilst MVP publishing tool is under development:
+# coronavirus_landing_page content items all have a sections_heading (eg Guidance and Support)
+# and a title field eg "Coronavirus (COVID-19): guidance and support". Both of these fields
+# are persisted in the collection publisher database, but they can still be updated in their relevant
+# yaml eg: https://github.com/alphagov/govuk-coronavirus-content/blob/master/content/coronavirus_landing_page.yml
+# If they get out of sync, this task can help.
+require_relative "../../app/services/coronavirus_pages/model_builder"
 
 namespace :coronavirus do
   desc "
-  Delete all coronavirus records.
+  Resync the title field and the sections_heading fields with the values stored in the yaml file.
   Usage
-  rake coronavirus:resync_database[dry_run_false]
+  rake coronavirus:resync_titles[slug]
   "
-  task :resync_database, [:dry_run] => [:environment] do |_task, args|
-    count = CoronavirusPage.count
-    slugs = CoronavirusPage.all.pluck(:slug)
-
-    if args.dry_run == "dry_run_false"
-      puts "Deleting #{count} coronavirus pages from the database: #{slugs}"
-      CoronavirusPage.destroy_all
-      CoronavirusPage.any? ? "Something went wrong, #{slugs} were not deleted." : "Done"
-    else
-      puts "This task is in dry run mode. It would have removed #{count} coronavirus pages from the database: #{slugs}"
-    end
+  task :resync_titles, [:slug] => [:environment] do |_task, args|
+    page = CoronavirusPage.find_by(slug: args.slug)
+    builder ||= CoronavirusPages::ModelBuilder.new(args.slug)
+    title = builder.title
+    sections_heading = builder.sections_heading
+    page.update!(title: title, sections_title: sections_heading)
```

Comment:
As `title` and `section_heading` defined on lines 18 and 19 are only used on line 20, this would be clearer:
```ruby
page.update!(
  title: builder.title,
  section_title: builder.section_title
)
```

Reply:
thanks!